src/JSInterop/Microsoft.JSInterop.JS/src/src/Microsoft.JSInterop.ts

```diff
@@ -237,6 +231,8 @@ module DotNet {
     });
 
     if (result instanceof Function) {
+      result = result.bind(lastSegmentValue);
```

Comment:
Can we have a test for this? Maybe an E2E test?

Reply:
Yes, but not in this repo